app/services/participants/base.rb

```diff
@@ -17,7 +17,7 @@ def call(params:)
 
     def call
       unless valid?
-        raise ActionController::ParameterMissing, errors.map(&:full_message)
+        raise ActionController::ParameterMissing, errors.map(&:message)
```

Comment:
this actually was a mistake from my part in the previous PR https://github.com/DFE-Digital/early-careers-framework/pull/1188

it actually changed the formatting of the error message.